versions/1.0/parsers/antlr4/WdlLexer.g4

```diff
@@ -1,6 +1,9 @@
-lexer grammar WdlLexer;
+lexer grammar V10WdlLexer;
```

Comment:
I would prefer the syntax of `Wdl<Version>Lexer` and `Wdl<Version>Parser`

Reply:
Do you like WdlV10Lexer or Wdl10Lexer? I would prefer WdlV1_0Lexer or Wdl1_0Lexer but it doesn't seem like Antlr likes underscores in these names.

Reply:
Could we do WdlV1Lexer for this case?